migrations/versions/0e9e0dffce7f_initial_schema_dump.py

```diff
@@ -0,0 +1,254 @@
+"""Initial schema dump.
```

Comment:
When I ran `alembic upgrade head`, I got an error. I think this happened because an upstream or downstream migration file was missing.
```
❯ alembic upgrade head                                                                                                                                      [428/1962]pudl_settings is being deprecated in favor of environment variables PUDL_OUTPUT and PUDL_INPUT. For more info see: https://catalystcoop-pudl.readthedocs.io/en/dev/dev
/dev_setup.html                                                                                                                                                       sqlite and parquet directories are no longer being used. Make sure there is a single directory named 'output' at the root of your workspace. For more info see: https:
//catalystcoop-pudl.readthedocs.io/en/dev/dev/dev_setup.html
pudl_settings is being deprecated in favor of environment variables PUDL_OUTPUT and PUDL_INPUT. For more info see: https://catalystcoop-pudl.readthedocs.io/en/dev/dev/dev_setup.html
sqlite and parquet directories are no longer being used. Make sure there is a single directory named 'output' at the root of your workspace. For more info see: https://catalystcoop-pudl.readthedocs.io/en/dev/dev/dev_setup.html
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/revision.py:223: UserWarning: Revision f3db07aa9714 referenced from f3db07aa971
4 -> a3686150733e (head), Add another output table is not present                                                                                                       util.warn(
Traceback (most recent call last):                                                                                                                                      File "/Users/bendnorman/mambaforge/envs/pudl-dev/bin/alembic", line 8, in <module>
    sys.exit(main())                                                                                                                                                               ^^^^^^                                                                                                                                                     File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/config.py", line 591, in main
    CommandLine(prog=prog).main(argv=argv)                                                                                                                              File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/config.py", line 585, in main
    self.run_cmd(cfg, options)
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/config.py", line 562, in run_cmd                                                  fn(
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/command.py", line 378, in upgrade
    script.run_env()
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/base.py", line 576, in run_env                                             util.load_python_file(self.dir, "env.py")
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/util/pyfiles.py", line 94, in load_python_file
    module = load_module_py(module_id, path)                                                                                                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/util/pyfiles.py", line 110, in load_module_py
    spec.loader.exec_module(module)  # type: ignore                                                                                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module                                                                                               File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed                                                                                          File "/Users/bendnorman/catalyst/pudl/migrations/env.py", line 79, in <module>
    run_migrations_online()                                                                                                                                             File "/Users/bendnorman/catalyst/pudl/migrations/env.py", line 73, in run_migrations_online
    context.run_migrations()                                                                                                                                            File "<string>", line 8, in run_migrations
File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/runtime/environment.py", line 868, in run_migrations
    self.get_context().run_migrations(**kw)                                                                                                                             File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/runtime/migration.py", line 609, in run_migrations
    for step in self._migrations_fn(heads, self):
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                        File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/command.py", line 367, in upgrade
    return script._upgrade_revs(revision, rev)                                                                                                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/base.py", line 450, in _upgrade_revs
    for script in reversed(list(revs))
                           ^^^^^^^^^^
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/revision.py", line 797, in iterate_revisions                               revisions, heads = fn(
                       ^^^                                                                                                                                              File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/revision.py", line 1424, in _collect_upgrade_revisions
    for rev in self._parse_upgrade_target(                                                                                                                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                              File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/revision.py", line 1216, in _parse_upgrade_target
    return self.get_revisions(target)                                                                                                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/revision.py", line 527, in get_revisions
    resolved_id, branch_label = self._resolve_revision_number(                                                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/revision.py", line 751, in _resolve_revision_number
    self._revision_map
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py", line 1113, in __get__                                    obj.__dict__[self.__name__] = result = self.fget(obj)
                                           ^^^^^^^^^^^^^^
  File "/Users/bendnorman/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/alembic/script/revision.py", line 227, in _revision_map                                   down_revision = map_[downrev]
                    ~~~~^^^^^^^^^
KeyError: 'f3db07aa9714'

Reply:
Also, I'm not sure all of the table creation statements are in this initial revision. I removed the two revision files and database file and generated an initial revision I got a ~2000 line file that creates all of the tables.

Reply:
hmm, let me regenerate the migrations then!